store/tikv/region_request.go

```diff
@@ -286,6 +288,7 @@ func (s *RegionRequestSender) SendReqCtx(
 
 			// TODO: Change the returned error to something like "region missing in cache",
 			// and handle this error like EpochNotMatch, which means to re-split the request and retry.
+			logutil.Logger(bo.ctx).Info("throwing pseudo region error due to region not founc in cache", zap.Stringer("region", &regionID))
```

Comment:
Can it print a lot? I don't think it's necessary.

Reply:
Maybe debug level is ok